cura/CuraApplication.py

```diff
@@ -2281,8 +2281,8 @@ def appHeight(self) -> int:
         return 0
 
     @pyqtSlot()
-    def deleteAll(self, only_selectable: bool = True) -> None:
-        super().deleteAll(only_selectable = only_selectable)
+    def deleteAll(self, only_selectable: bool = True, clear_all:bool = False) -> None:
```

Comment:
```suggestion
    def deleteAll(self, only_selectable: bool = True, clear_all:bool = False) -> None:
```

We should guard against bool traps here. You can do that by by adding a param that eats up all unnamed arguments.